lib/managers/files.js

```diff
@@ -575,6 +575,43 @@ Files.prototype.deletePermanently = function(fileID, callback) {
 	this.client.del(apiPath, null, this.client.defaultResponseHandler(callback));
 };
 
+/*
+ * Restores an item that has been moved to the trash. Default behavior is to
+ * restore the item to the folder it was in before it was moved to the trash.
+ * If that parent folder no longer exists or if there is now an item with the
+ * same name in that parent folder, the new parent folder and/or new name will
+ * need to be included in the request.
+ *
+ * API Endpoint: '/files/:fileID'
+ * Method: POST
+ *
+ * @param {string} fileID - The ID of the file to remove metadata from
+ * @param {?string} [name] - The new name for this item
+ * @param {string} [parentFolderID] - The new parent folder for this item
+ * @param {Function} callback - Called with item information if successful, error otherwise
+ * @returns {void}
+ */
+Files.prototype.restoreItem = function(fileID, name, parentFolderID, callback) {
```

Comment:
See above, but I think we should rename this to restoreFromTrash to be more informative